projects/gnu-tools/monthly-report-2020-12-14.md

```diff
@@ -0,0 +1,79 @@
+# Software TG monthly meeting - CORE-V GNU Tool Chain (December 2020)
+
+## Overview
+
+### Key activities
+
+* Added multiply accumulate and general ALU operations support to GNU Binutils incl. full testing - in OpenHW repos (Embecosm)
+* Added post-increment and register-indexed load/store support to GNU Binutils incl. full testing - not yet in OpenHW repos (UOB/ETHZ)
+* Added general ALU operations support to GNU GCC, not tested as awaiting hardware - not yet in OpenHW repos (Embecosm)
+* Started adding immediate branching instructions support to support to GNU Binutils (UOB/ETHZ)
+* Added new -march option: xcorevalu (supports the general ALU operations subset) and xcorevpostinc (supports the post-increment and register-indexed load/store operations subset) in binutils and xcorevalu to GCC (UOB/ETHZ and Embecosm)
+* Updated documentation to include all CORE-V extensions (found at: https://github.com/openhwgroup/core-v-docs/tree/master/cores/cv32e40p/user_manual)
```

Comment:
Formal policy proposal to support Vendor Specific GCC targets has been submitted to RISC-V Toolchain & Runtimes subcommittee by Jeremy Bennett.  OpenHW Group members who are also RISC-V members are encouraged to engage with this review process.

(Realise this has changed, since I started writing, but it belongs up here as well. Note there are two things going on in RISC-V:

- formal policy proposal to support vendor specific extensions
- proposal to create the psABI task group)